xonsh/pyghooks.py

```diff
@@ -281,7 +282,7 @@ def __init__(self, style_name='default'):
         style_name : str, optional
             The style name to initialize with.
         """
-        self.trap = {}  # for custom colors
+        self.trap = builtins.__xonsh_env__.get('PROMPT_TOOLKIT_STYLES') or {}
```

Comment:
You are right. The name is stupid. I just recreated the functionality from the old color system.

I think we could even completely remove the environment variable. The shell.styler hold state of the styles anyway. So the same thing can be accomplished without the environment variable. I will make some changes. Hope you like the more :)